shared/chat/conversation/messages/wrapper/container.js

```diff
@@ -100,10 +114,11 @@ export default compose(
   }),
   lifecycle({
     componentDidUpdate: function (prevProps: Props & {_editedCount: number}) {
-      if (this.props.measure &&
+      if (this.props.measure && (
         (this.props._editedCount !== prevProps._editedCount) ||
-        (this.props.isFirstNewMessage !== prevProps.isFirstNewMessage)
-      ) {
+        (this.props.isFirstNewMessage !== prevProps.isFirstNewMessage) ||
+        (this.props.timestamp !== prevProps.timestamp)
+      )) {
         this.props.measure()
```

Comment:
🐑 🐑 this.props.measure && this.props.measure(). then you don't need the && ( ... || ... || ... )

double sheep - totes optional